### PR TITLE
Updated landing page (first iteration)

### PR DESCRIPTION
**Pull request by IgnaciodeNuevo**
@iesteban 

- Need to review the login after merging (it crashes when you click on it)
- Also it crashes when trying to change language by clicking on the language flag

### CODE REVIEW

File: semillas_backend/templates/base.html [hunk 1]
```diff
@@ -13,165 +13,162 @@
...
-                <a class="nav-link" href="{% url 'home' %}"><i class="fa fa-home" aria-hidden="true"></i> Home</a>
-              </li>
-              <li class="nav-item">
-              <a class="nav-link" href="{% url 'landing:people' %}"><i class="fa fa-users" aria-hidden="true"></i> {% trans "Quiénes Somos" %}</a>
+
+    <!-- Header -->
+      <header class="page-head--masthead">
+        <div class="wrapper">
+          <nav id="nav" class="site-nav  clearfix">
+            <a href="/"><img class="site-nav__logo" src="/static/images/logo.png" alt="Semilla Social Logo"></a>
```
Comment (iesteban) [inline semillas_backend/templates/base.html]: We need to use here the {% static tag

File: semillas_backend/static/css/layout/_header.scss [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    @media screen and (min-width: $bpMedium) {
+      .page-head--masthead {
+          background-image: url(/static/images/masthead-medium.jpg);
+      }
+    }
+
+    @media screen and (min-width: $bpLarge) {
+      .page-head--masthead {
+          background-image: url(/static/images/masthead-large.jpg);
```
Comment (iesteban) [inline semillas_backend/static/css/layout/_header.scss]: For this I would try to use relative paths... maybe... 

Check out how alpha stores its statics: 
The browser is looking for it (failing) here:
https://s3.amazonaws.com/static/images/masthead-large.jpg

While it is stored here:
https://s3.amazonaws.com/semillaalpha/images/masthead-large.jpg

**Comment by IgnaciodeNuevo** [comment]
Fixed images path, moved to Amazon S3, check if it works ok @iesteban

File: semillas_backend/static/css/layout/_header.scss [hunk 3]
```diff
@@ -18,13 +18,13 @@
...
       .page-head--masthead {
-          background-image: url(/static/images/masthead-medium.jpg);
+          background-image: url(https://s3.amazonaws.com/semillaalpha/images/masthead-medium.jpg);
       }
     }
 
     @media screen and (min-width: $bpLarge) {
       .page-head--masthead {
-          background-image: url(/static/images/masthead-large.jpg);
+          background-image: url(https://s3.amazonaws.com/semillaalpha/images/masthead-large.jpg);
```
Comment (IgnaciodeNuevo) [inline semillas_backend/static/css/layout/_header.scss]: It seams to be ok

**Pull request merged**